src/interfaces.d.ts

```diff
@@ -290,7 +290,7 @@ export interface WNode {
 /**
  * union type for all possible return types from render
  */
-export type DNode = HNode | WNode | string | null;
+export type DNode<P extends WidgetProperties = WidgetProperties, C extends WidgetProperties = WidgetProperties> = HNode | WNode<P, C> | string | null;
```

Comment:
Okay, so this causes errors when add `v(...)` generate nodes (`HNode`) as it cannot infer `C` (because the type is not used in by the `HNode`